deployment/keystone/changeset/internal/deploy.go

```diff
@@ -1079,3 +1046,28 @@ func configureForwarder(lggr logger.Logger, chain deployment.Chain, contractSet
 	}
 	return opMap, nil
 }
+
+func GetDefaultCapConfig(capability capabilities_registry.CapabilitiesRegistryCapability) *capabilitiespb.CapabilityConfig {
```

Comment:
This is only used by tests, right? If so, it should exist in the test subdirectory.